lib/ex_doc/translations.ex

```diff
@@ -0,0 +1,304 @@
+defmodule ExDoc.Translations do
+
+  @locale_spanish_international "es"
+  @locale_spanish_spain "es-ES"
+  @locale_galician_spain "gl-ES"
+
+  def translate(@locale_spanish_international, "API Reference"), do: "Referencia de la API"
+  def translate(@locale_spanish_international, "Built using"), do: "Construido usando"
+  def translate(@locale_spanish_international, "Cancel search"), do: "Cancelar búsqueda"
+  def translate(@locale_spanish_international, "Collapse/expand sidebar"), do: "Contraer/expandir barra lateral"
+  def translate(@locale_spanish_international, "Cover"), do: "Cubierta"
+  def translate(@locale_spanish_international, "Display keyboard shortcuts"), do: "Mostrar combinaciones de teclas"
+  def translate(@locale_spanish_international, "Example"), do: "Ejemplo"
+  def translate(@locale_spanish_international, "Examples"), do: "Ejemplos"
+  def translate(@locale_spanish_international, "Exceptions"), do: "Excepciones"
+  def translate(@locale_spanish_international, "Function"), do: "Función"
+  def translate(@locale_spanish_international, "Functions"), do: "Funciones"
+  def translate(@locale_spanish_international, "Link"), do: "Enlace"
+  def translate(@locale_spanish_international, "Link to this "), do: "Enlace a esto "
+  def translate(@locale_spanish_international, "Link to this section"), do: "Enlace a esta sección"
+  def translate(@locale_spanish_international, "Mix Tasks"), do: "Tareas de Mix"
+  def translate(@locale_spanish_international, "Modules"), do: "Módulos"
+  def translate(@locale_spanish_international, "Option"), do: "Opción"
+  def translate(@locale_spanish_international, "Options"), do: "Opciones"
+  def translate(@locale_spanish_international, "Page not found"), do: "Página no encontrada"
+  def translate(@locale_spanish_international, "Pages"), do: "Páginas"
+  def translate(@locale_spanish_international, "Search"), do: "Buscar"
+  def translate(@locale_spanish_international, "Search..."), do: "Buscar..."
+  def translate(@locale_spanish_international, "Summary"), do: "Resumen"
+  def translate(@locale_spanish_international, "Table Of Contents"), do: "Tabla de Contenidos"
+  def translate(@locale_spanish_international, "Table Of contents"), do: "Tabla de contenidos"
+  def translate(@locale_spanish_international, "This is deprecated"), do: "Esto quedó obsoleto"
+  def translate(@locale_spanish_international, "Toggle night mode"), do: "Cambiar al modo noche"
+  def translate(@locale_spanish_international, "Type"), do: "Tipo"
+  def translate(@locale_spanish_international, "Types"), do: "Tipos"
+  def translate(@locale_spanish_international, "View Source"), do: "Ver Fuente"
```

Comment:
```suggestion
  def translate(@locale_spanish_international, "View Source"), do: "Ver Código"
```
Or "Ver Código Fuente"

Reply:
👍